src/main/scala/tilelink/Xbar.scala

```diff
@@ -67,6 +70,7 @@ class TLXbar(policy: TLArbiter.Policy = TLArbiter.roundRobin)(implicit p: Parame
     override def circuitIdentity = outputs.size == 1 && inputs.size == 1
   }
 
+  // implementation
```

Comment:
This is not a good code comment. Code comments should point out something unusual that cannot be gleamed from the code itself.